pack.sh

```diff
@@ -37,11 +39,20 @@ node --experimental-worker $(which $ROSETTA) \
   $(cat $TMPDIR/jsii.txt)
 
 # Jsii packaging (all at once using jsii-pacmak)
+# execute monocdk packaging in parallel
 echo "Packaging jsii modules" >&2
 $PACMAK \
   --verbose \
   --rosetta-tablet samples.tabl.json \
-  $(cat $TMPDIR/jsii.txt)
+  $(cat $TMPDIR/jsii.txt) &
+
+
+echo "Packaging mono jsii modules" >&2
+$PACMAK \
+  --verbose \
+  $(cat $TMPDIR/jsii-monolithic.txt) &
+
+wait
```

Comment:
I recently wanted to do the same and ran into the issue that `wait`:

* Will wait until all jobs are done 
* If you ctrl-c it while waiting, the shell script will stop but the background jobs will continue running in the background
* Will NOT exist with an error code in this invocation style (you need to write `wait %1 %2` to have it exist with an error code if one of the jobs fails).

Which led me to conclude that `wait` is just too sharp an edge that needs to much handholding to do what we want.

I've used `make` instead: https://github.com/aws/cdk-ops/pull/768